src/classes/PS_ElevateService.cls

```diff
@@ -0,0 +1,94 @@
+/*
+    Copyright (c) 2020, Salesforce.org
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+      
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2020
+* @group Elevate Payment Services
+* @description Contains Elevate payment services methods 
+*/
+public with sharing class PS_ElevateService {
```

Comment:
There are common Payment Services methods that will be used by RD2 and probably other features, but currently they are contained in `GE` classes. I *copied* the common methods into a new Apex class. I have not extracted them from the `GE` classes, but I would suggest they are updated to use the common code in another WI. 

This new controller should be added to the required profiles' enabled Apex classes.

Reply:
@RadSFDeveloper Because this class queries the `Organization` object, which is a System object, I think the class should be defined as either `without` or `inherited sharing` to avoid the same issue we ran into with the ApexAsyncJob system object. I will note that other classes that query this object are mostly implied inherited. There is one class that uses `with sharing`, so I suspect it would work even if left as is.

Reply:
Depending on timing, I wonder if we (Honeygems) might want to put in a PR as a child of this work to pivot to the new PS-prefixed variants. At the time we initially built this there wasn't enough intel to know if we were really going to share concepts like "isElevateCustomer" outside of Gift Entry, but since it is now clear that we will and are sharing these functions, I agree that this switch makes sense. But if we gather all that work into this PR we can solve for it now, rather than waiting until this work merges when we will inevitably be focused on other things.

Reply:
@patrick-yan-sf , feel free to create branches off the `feature/RD2` to incorporate work by Honeygems related to Elevate Payment Services based off the common classes. The `feature/RD2` will be main branch for Elevate commitment creation work.